src/board/gridBoard.js

```diff
@@ -0,0 +1,33 @@
+// gridBoard.j
+var inherits = require('util').inherits;
+var Board = require("../board/board.js");
+
+function GridBoard(width, height) { 
+  Board.call(this);
+  for (var i = 0; i < height; i++) { 
+    this.spaces[i] = [];
+  }
+  this.width = width;
+  this.height = height;
+};
+
+inherits(GridBoard, Board);
+
+GridBoard.prototype.getSpace = function(x, y) { 
+  return this.spaces[x][y];
+};
+
+GridBoard.prototype.getSpacePosition = function(space) { 
+
+  for (var x = 0; x < this.width; x++) {
+    for (var y = 0; y < this.height; y++) { 
```

Comment:
Probably not worth adding in today but something to think about. You can just `npm install loadsh` i think